### DISCUSSION TITLE
how to hide new create button?

### DISCUSSION THREAD

**Discussion by phamduc1823** [filamentphp/filament#7919] [General]
how can i disable the display of the new create button? i want to show the user only the table and don't want to create a new one

**Comment by gilr** [comment]
Remove or comment the CreateAction in your Filament/Resources/<Model>Resource/Pages/List<Model>.php

```
    protected function getHeaderActions(): array
    {
        return [
        //       Actions\CreateAction::make(),
        ];
    }

```

**Reply by phamduc1823** [reply]
thankssssss bro
